fpmsyncd/routesync.cpp

```diff
@@ -563,7 +1426,20 @@ void RouteSync::onMsgRaw(struct nlmsghdr *h)
             (size_t)NLMSG_LENGTH(sizeof(struct ndmsg)));
         return;
     }
-    onEvpnRouteMsg(h, len);
+
+    switch (getEncapType(h))
+    {
+        case NH_ENCAP_SRV6_ROUTE:
+            onSrv6SteerRouteMsg(h, len);
+            break;
+        case NH_ENCAP_SRV6_LOCAL_SID:
+            onSrv6LocalSidMsg(h, len);
+            break;
+        default:
+            onEvpnRouteMsg(h, len);
```

Comment:
it may be good to have a comment here to mention why default case handles onEvpnRouteMsg. I understand you are mainting functionality as before switch case, but for future reading it may be confusing without supporting comment

Reply:
I added a comment, thanks for the suggestion!